src/main/java/domain/model/Ladder.java

```diff
@@ -0,0 +1,36 @@
+package domain.model;
+
+import domain.vo.Height;
+import domain.vo.Width;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ladder {
+    private final Height height;
+    private final Width width;
+    private final List<Layer> layers;
```

Comment:
```suggestion
    private final Layers layers;
```
일급 컬렉션을 사용하지 않은 이유가 있나요??

Reply:
Layers == Ladder 이라고 생각했습니다. 그리고 Layer에서는 제약 사항(옆 라인의 다리가 존재한다면 현 라인은 존재할 수 없다.)가 있어 일급 컬렉션이 의미가 있지만 단순하게 Layer가 모여있는 Layers에 따로 비즈니스 로직이 없어 따로 만들지 않았습니다.

Reply:
음.. 그렇게 판단할 수 도 있겠네요 ㅎㅎ

만약 우연히 모든 Layer가 다리가 없는 같은 Layer로 생성된다면, 어떨게 될까요?? 😄

주어진 요구사항을 지키는 것도 무척 중요하지만, 숨어 있는 제약 사항을 찾고 이를 논의하고 반영하는 것도 중요하다고 생각해요!

Reply:
만약 우연히 모든 Layer가 다리가 없는 같은 Layer로 생성된다면, 어떨게 될까요?? 😄
--> 이 부분이 이해가 잘 되지 않습니다🥲 다리가 없는 같은 Layer라면 그냥 일자인 사다리가 만들어질 것이고 혹시 그게 문제가 된다는 말씀이신가요?? 만약 이게 맞다면 어떤 문제가 발생하는지 모르겠습니다ㅠㅠ 
제 생각에는 일자 사다리가 생성 되어도 랜덤하게 만들어졌기 때문에 그 자체로 의미가 있는 것 같습니다! 혹시 제가 잘못 이해 했는지 확인 부탁 드리겠습니다!

Reply:
네 이해하신 부분이 맞습니다 ㅎㅎ

개인적인 생각일 수 있겠지만.. 모두 일자라면 그건 더 이상 사다리가 아니라고 생각해서요!

모두 랜덤이라서 문제가 없다고  생각할 수 있겠지만, 이 부분이 문제일 경우 놓치고 지나간다면 어떻게 될까요??
이러한 부분은 실제 문제가 발생하기 전까지 파악하기가 어렵기 때문에, 미리 파악하고 해당 부분을 어떻게 처리할 지 팀에서(지금은 페어와) 논의해보면 좋을 것 같아요!

문제가 아니라고 생각된다면 넘어가셔도 될 것 같습니다 😅